uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/sickness-leave-request-instance_test.js

```diff
@@ -31,93 +31,31 @@ define([
       }
     ]));
 
-    describe('create()', function () {
-      beforeEach(function () {
-        instance = SicknessRequestInstance.init({}, false);
-        promise = instance.create();
-      });
-
-      afterEach(function () {
-        //to excute the promise force an digest
-        $rootScope.$apply();
-      });
-
-      it('calls equivalent API method', function () {
-        expect(LeaveRequestAPI.create).toHaveBeenCalledWith(jasmine.any(Object), 'sick');
-      });
-
-      it('calls toAPI method', function () {
-        expect(instance.toAPI).toHaveBeenCalled();
-      });
-
-      describe('id field', function() {
-        it('is not appended to instance after API returns data', function() {
-          expect(instance.id).not.toBeDefined();
-        });
-
-        it('is appended to instance after API returns data', function() {
-          promise.then(function () {
-            expect(instance.id).toEqual(jasmine.any(String));
-          });
-        });
-      });
-    });
-
-    describe('isValid()', function () {
-      beforeEach(function () {
-        instance = SicknessRequestInstance.init({}, false);
-        promise = instance.isValid();
-      });
-
-      afterEach(function () {
-        $rootScope.$apply();
-      });
-
-      it('calls equivalent API method', function () {
-        expect(LeaveRequestAPI.isValid).toHaveBeenCalledWith(jasmine.any(Object), 'sick');
-      });
-
-      it('calls toAPI method', function () {
-        expect(instance.toAPI).toHaveBeenCalled();
-      });
-    });
-
-    describe('update()', function () {
-      beforeEach(function () {
+    describe('default attributes', function() {
+      beforeEach(function() {
         instance = SicknessRequestInstance.init({}, false);
-        promise = instance.update();
-      });
-
-      afterEach(function () {
-        $rootScope.$apply();
       });
 
-      it('calls update api method with the return value of toAPI method', function () {
-        expect(LeaveRequestAPI.update).toHaveBeenCalledWith(jasmine.any(Object), 'sick');
+      it('initializes sickness reason', function() {
+        expect(instance.sickness_reason).toBeNull();
```

Comment:
Right now its confusing, because you are saying that reason initializes and testing toBeNull, better use the following?
`expect(instance.sickness_reason).not.toBeUndefined();`